projects/plugins/jetpack/jetpack.php

```diff
@@ -116,13 +116,11 @@
 	 * @since 7.2.0
 	 */
 	function jetpack_admin_unsupported_wp_notice() {
-		wp_admin_notice(
-			esc_html__( 'Jetpack requires a more recent version of WordPress and has been paused. Please update WordPress to continue enjoying Jetpack.', 'jetpack' ),
-			array(
-				'type'        => 'error',
-				'dismissible' => true,
-			)
-		);
+		?>
+		<div class="notice notice-error is-dismissible">
+			<p><?php esc_html_e( 'Jetpack requires a more recent version of WordPress and has been paused. Please update WordPress to continue enjoying Jetpack.', 'jetpack' ); ?></p>
+		</div>
+		<?php
```

Comment:
Would it be worth doing something like this to use the new function when it's available?
```suggestion
		$msg = __( 'Jetpack requires a more recent version of WordPress and has been paused. Please update WordPress to continue enjoying Jetpack.', 'jetpack' );
		// This is a check for old versions of WordPress. Maybe so old that this function doesn't exist.
		if ( is_callable( 'wp_admin_notice' ) ) {
			wp_admin_notice(
				esc_html( $msg ),
				array(
					'type'        => 'error',
					'dismissable' => true,
				)
			);
		} else {
			?>
			<div class="notice notice-error is-dismissible">
				<p><?php echo esc_html( $msg ); ?></p>
			</div>
			<?php
		}
```
(and similarly for the other one)

Reply:
This seems a bit overkill to me for just a notice. I'd keep things simple for now.